css/index.css

```diff
@@ -99,8 +121,11 @@ p {
     display: flex;
     flex-wrap: wrap;
     justify-content: space-evenly;
+
+
 }
 
+
```

Comment:
When you get to styling your boxes, parent/child relationships are key to properly setting the boxes up.  You want to use the parent styles so that the children inherit them.  This requires some work inside the CSS file and the index.html.